controller/decorator/controller.go

```diff
@@ -412,6 +412,8 @@ func (c *decoratorController) syncParentObject(parent *unstructured.Unstructured
 		return err
 	}
 
+	glog.V(4).Infof("DecoratorController %v: observedChildren %+v", c.dc.Name, observedChildren)
```

Comment:
Do you think this log line will be needed long term, or was this more for debugging this particular PR? If you think it'll be useful long term, we should raise it to something like [log level 6](https://metacontroller.app/guide/troubleshooting/#log-levels).

Reply:
I'm pretty sure it's not useful outside of this context. I'll get rid of it.